content/intel-me.md

```diff
@@ -0,0 +1,38 @@
+---
+title: What is the Intel Management Engine?
+description: >
+    This article describes the Intel Management Engine, scenarios in which the IME is disabled, and how to manually disable on systems running coreboot.
+keywords:
+  - Firmware
+  - Intel ME
+  - Management Engine
+  - Disable Intel ME
+  - IME
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+System76 laptops and desktops with Open Firmware will typically have the Intel Management Engine (IME) disabled, unless doing so breaks functionality. Machines running Open Firmware with coreboot can disable the IME manually.
+
+## What is the Intel Management Engine?
+
+The [Intel Management Engine](https://en.wikipedia.org/wiki/Intel_Management_Engine) is a proprietary, mostly undocumented, firmware system that provides many extraneous features that are generally not usable or useful to our users, with multiple known vulnerabilities that compromise the entire system.
+
+## What is System76 Open Firmware?
+
+System76 Open Firmware is an open source distribution of firmware utilizing coreboot, EDK2, and System76 firmware applications. System76 Open Firmware can disable the IME, [among other features](/articles/transition-firmware).
+
+## Is the IME Disabled On My System76 PC?
+
+For Open Firmware systems, the IME is typically disabled by default unless doing so would break functionality (such as suspend/resume). System76 maintains a list of machines that ship with Open Firmware [in this article](/articles/open-firmware-systems/).
```

Comment:
To better answer the question, you could also mention that the firmware menu -> Firmware Configuration Information page will show whether the IME is enabled or disabled.

Example picture (you don't need to use this, but just to show how obvious it is on that page):

![me-enabled](https://user-images.githubusercontent.com/7199422/178847130-e3e4164e-9b53-4f65-95fa-de434180aef7.jpg)

Reply:
Updated these instructions. Thanks for the heads up.